### ISSUE TITLE
Will this continue as source of the Faker library? #1

### ISSUE THREAD

**Issue by mmahalwy** [faker-js/faker.js#24]
@Geertvdc asked in https://github.com/withshepherd/faker.js/issues/1#issue-1094203363:

```
I noticed this morning that the original faker repo and npm package were all removed or emptied (https://github.com/Marak/Faker.js). Looking for a fork or alternative I bumped into this.

Will you continue building / supporting faker.js? Do you need any help? Could you elaborate on the plans? Did you know Marak?

I think Faker.js by Marak was a well used library and it would be a shame if it would go away. I would love helping out in getting this back up if you need any help.
```

Continuing conversation here

**Comment by prisis** [comment]
I will try to port old PRs from web archive to this repo

**Comment by damienwebdev** [comment]
@prisis just to ensure that we respect the original contributor of the PR, can you ref the PR (or the author here) so that we can ping them and have them submit it themselves if they want to?

**Comment by prisis** [comment]
Yeah can do it

**Comment by G1itcher** [comment]
This will probably be an unpopular opinion, but it seems a little disrespectful to do this?

I believe Marak took his project down specifically to protest exploitation of open source by large companies after losing almost everything in a fire? To nullify his protest like this seems a little wrong to me.

**Comment by zachariahtimothy** [comment]
> This will probably be an unpopular opinion, but it seems a little disrespectful to do this?
> 
> I believe Marak took his project down specifically to protest exploitation of open source by large companies after losing almost everything in a fire? To nullify his protest like this seems a little wrong to me.

Is certainly an interesting take. Was it respectful of him to cost our team a couple hours of collective productivity by releasing a new version with no code and leaving us scratching our heads why the build was broken? 
It was his prerogative to do that but this is the nature of open source. This library is necessary and utilized by many. Perhaps he should have been appropriately compensated, but to say nobody is allowed to maintain a fake data library because the original author was unsponsored does not make sense to me.

**Comment by G1itcher** [comment]
I acknowledge that it's a complicated subject that can't be easily slotted on either side of anyone's morality. Just thought it was worth expressing.

**Comment by zachariahtimothy** [comment]
Best thing we can do to avoid this type of thing in the future is to our employers to do more sponsorship! Libraries like this save many many hours of development so why not give even a couple hundred bucks?

**Comment by prisis** [comment]
Hey @MilosPaunovic and @gustawdaniel-acaisoft do you like to work with us on faker.js v6?

**Comment by MilosPaunovic** [comment]
@prisis Hmm... You mean to discard the `community-faker` altogether?

I'm up for it, as I think it will benefit the open source community more than to have multiple versions of, basically, the same thing. Did you have anything particular on your mind?

**Comment by prisis** [comment]
@MilosPaunovic Yes :), we would give to maintainer rights if the other team member are fine with it and if you want it :) Nothing particular in mind just to work together :)
